src/wkok/openai_clojure/azure/core.clj

```diff
@@ -0,0 +1,34 @@
+(ns wkok.openai-clojure.azure.core
```

Comment:
Please add `^:no-doc` metadata so the codox document generation skips the `core` namespace

Reply:
done